_pytest/python.py

```diff
@@ -743,6 +743,48 @@ def __init__(self, function, fixtureinfo, config, cls=None, module=None):
         self._ids = set()
         self._arg2fixturedefs = fixtureinfo.name2fixturedefs
 
+    def parameterize(self, argnames, argvalues, indirect=False, ids=None,
```

Comment:
If we are to include this, I prefer if you would just create an actual alias in the class:

```
parameterize = parametrize
```

While mentioning in the docs that `parameterize` is a valid alias; we don't want to have the documentation duplicated.

Reply:
That does seem like a more elegant solution. Should it be placed below the `parametrize` definition with a comment stating it's an alias?

Reply:
Sounds good. 

Please add a test which uses the new spelling as well.

Reply:
I'm having trouble getting the test to work, so far I have this:
``` python
def test_parameterize(self, testdir):
    testdir.makepyfile("""
        import pytest
        @pytest.mark.parameterize("foo", [1, 2, 3])
        def test_foo(foo):
            assert foo in [1, 2, 3]
    """)
    reprec = testdir.inline_run()
    reprec.assertoutcome(passed=3)
```
And it complains about `fixture 'value' not found`.
If I use `parametrize` instead, it does work.
What am I doing wrong? 🤔

Reply:
`value`? Perhaps you mean `foo`? If that's the case I think you also need to handle that name here:

https://github.com/pytest-dev/pytest/blob/3bc7ced97ae2406916f4321ed63d2a4376e4a1aa/_pytest/fixtures.py#L1054

Perhaps something like: `getattr(metafunc.function, 'parametrize', None) or getattr(metafunc.function, 'parameterize', None)` (ugh).

Otherwise please post the full error message.

Reply:
Yes, I meant `foo`. 😅 (I was also testing on a separate file where I used `value` instead)

I changed it but still got the same error, posting the full error below:
<details>
<summary> Error </summary>

```
==================================== ERRORS ====================================
__________________________ ERROR at setup of test_foo __________________________
file /tmp/pytest-of-alanbato/pytest-30/test_parameterize0/test_parameterize.py, line 2
  @pytest.mark.parameterize("foo", [1, 2, 3])
  def test_foo(foo):
E       fixture 'foo' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/tmp/pytest-of-alanbato/pytest-30/test_parameterize0/test_parameterize.py:2
```

</details>

Reply:
You applied the change I mentioned in `fixtures.py` right? Hmm so it must be missing somewhere else... not sure where, a quick search didn't turn up anything obvious.

But I suggest for you to not spend too much time on this until we decide for sure if we want to merge this or not. 😅